addon/components/step-manager/step.js

```diff
@@ -2,7 +2,7 @@ import Component from '@ember/component';
 import { get, set } from '@ember/object';
 import { isPresent } from '@ember/utils';
 import { assert } from '@ember/debug';
-import { computed } from '@ember-decorators/object';
+import { computed } from '@ember/object';
 import { tagName } from '@ember-decorators/component';
```

Comment:
@jasonmevans I think you'd originally moved this import as well, but it is not part of the polyfill. I fixed that while rebasing.